alvr/client_openxr/Cargo.toml

```diff
@@ -160,26 +163,33 @@ value = "3,6DoF"
 [[package.metadata.android.uses_permission]]
 name = "com.picovr.permission.EYE_TRACKING"
 [[package.metadata.android.application.meta_data]]
-name = "pvr.app.type"
-value = "vr"
-[[package.metadata.android.application.meta_data]]
-name = "picovr.software.eye_tracking"
-value = "true"
-[[package.metadata.android.application.meta_data]]
 name = "eyetracking_calibration"
 value = "true"
 [[package.metadata.android.application.meta_data]]
 name = "handtracking"
 value = "1"
+[[package.metadata.android.application.meta_data]]
+name = "picovr.software.eye_tracking"
+value = "1"
+[[package.metadata.android.application.meta_data]]
+name = "pvr.app.type"
+value = "vr"
+[[package.metadata.android.application.meta_data]]
+name = "pvr.display.orientation"
+value = "180"
+[[package.metadata.android.application.meta_data]]
+name = "pvr.sdk.version"
+value = "OpenXR"
+[[package.metadata.android.application.meta_data]]
+name = "pxr.sdk.version_code"
+value = "5900"
```

Comment:
Huh?

Reply:
This is the magic patch that allowed me to upload the build. the value is the minimum firmware version